package.json

```
@@ -36,7 +36,10 @@
     "docker:test:init": "docker-compose -f docker-compose.tests.yml run -u root --rm tests bash change_volumes_owner.sh",
     "docs:serve": "docsify serve ./docs",
     "ensure-env": "cross-env DEBUG=fcc:* node ./tools/scripts/build/ensure-env.js",
-    "e2e": "./cypress/run-e2e.sh",
+    "e2e:dev:watch": "start-test develop 'http://localhost:3000/status/ping|8000' cypress:dev:watch",
+    "e2e:dev:run": "start-test develop 'http://localhost:3000/status/ping|8000' cypress:dev:run",
+    "e2e:prd:watch": "npm run build && start-test start 'http://localhost:3000/status/ping|8000' cypress:dev:watch",
+    "e2e:prd:run": "npm run build && start-test start 'http://localhost:3000/status/ping|8000' cypress:dev:run",
```

```suggestion
    "e2e:dev:watch": "start-test develop ':3000/status/ping|8000' cypress:dev:watch",
    "e2e:dev:run": "start-test develop ':3000/status/ping|8000' cypress:dev:run",
    "e2e:prd:watch": "npm run build && start-test ':3000/status/ping|8000' cypress:dev:watch",
    "e2e:prd:run": "npm run build && start-test ':3000/status/ping|8000' cypress:dev:run",
```
We can make it a little more concise by using the 'start' convention and the url shortcuts.  Is it still clear what's going on?